build.xml

```diff
@@ -10,19 +11,30 @@
 
 	<property name="version" value="0.0.0" />
 
-	<property name="build.dir" value="./bin" />
-	<property name="build.dir.compile" value="${build.dir}/classes" />
+	<property name="build.dir" value="./bin/main" />
```

Comment:
It was a template that had it in `/classes`, I keep them in `/main` because there are test binaries placed into `/bin/test` so when packaging its simpler to pull them out (although I don't think I tested if the `package` target works properly...).